myst.yml

```diff
@@ -8,14 +8,23 @@ project:
   authors: []
   github: https://github.com/pyopensci/peer-review-metrics
   # bibliography: []
+
 site:
+  nav:
```

Comment:
My goal is to control both top level nav and side bar nav but i can't seem to do that. this config gives me this - it just ignores the top nav altogether and tries to create it's own sidebar nav

<img width="1061" alt="Screenshot 2024-08-09 at 8 07 26 PM" src="https://github.com/user-attachments/assets/f0f6cd20-a6bc-402c-8e77-be2e4b07dcf3">

@agoose77 i wondered if anything popped out for you here as to why this isn't working (i also asked in discord).

Reply:
Hi @lwasser - Here's where the mystmd.org top `nav` is defined; it includes links and `children` drop-downs: https://github.com/jupyter-book/mystmd.org/blob/main/content/myst.yml. 

You must use relative URLs, not files (my suggested change demonstrates how to do this here). If you do want to have direct file downloads, you can put them in actions, something like:
```
  actions:
    - title: Current Reviews Data Download
      url: current-reviews-data.ipynb
```

Regarding the sidebar - you can customize the order and contents. If you run `myst init --write-toc`, it will put a table of contents `toc` in your `myst.yml`. You can then move things around or remove unwanted pages. (However, pages linked in `nav` must be part of the `toc`.)